src/arrayMethodSort.js

```diff
@@ -4,8 +4,30 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function (compareFunction = ....) {
```
Consider using the default parameter here